templates/js-template-link/frameworks/runtime-src/Classes/AppDelegate.cpp

```diff
@@ -44,7 +43,7 @@
 
 USING_NS_CC;
 
-AppDelegate::AppDelegate() : Application("js-test")
+AppDelegate::AppDelegate(int width, int height) : Application("Cocos Runtime", width, height)
```

Comment:
`Cocos Runtime` is not suitable here.

Reply:
I think so, hard coded the application name is a bad choice, might we can add a macro
```
// modify it to change application name
#define COCOS_APPLICATION_NAME "Cocos Game"
```
for further, we can modify it by cocos console, for example change default value to `HelloJavaScript`, change it to `hellowrold` when the identifier is `org.cocos2d.helloworld`